src/main/java/svc/data/citations/ViolationDAO.java

```diff
@@ -57,6 +60,47 @@ public Violation getViolationDataById(int violationId)
 		return violations;
 	}
 	
+	public boolean insertDemoViolations(List<Violation> violations){
```

Comment:
Even though this is only used for demo, This class is the ViolationDAO. So I would just omit the word Demo from anything in here because it's not relevant to the actual function